src/effects/effectmanifestparameter.h

```diff
@@ -202,21 +204,22 @@ class EffectManifestParameter {
     virtual const double& getDefault() const {
         return m_default;
     }
-    virtual void setDefault(const double& defaultValue) {
-        m_default = defaultValue;
-    }
 
     virtual const double& getMinimum() const {
         return m_minimum;
     }
-    virtual void setMinimum(const double& minimum) {
-        m_minimum = minimum;
-    }
 
     virtual const double& getMaximum() const {
         return m_maximum;
     }
-    virtual void setMaximum(const double& maximum) {
+
+    virtual void setRange(const double& minimum, const double& defaultValue, const double& maximum) {
```

Comment:
Good idea to combine these into one function to assure all values are valid. :+1: